file/file_prefetch_buffer.cc

```diff
@@ -635,6 +644,9 @@ bool FilePrefetchBuffer::TryReadFromCache(const IOOptions& opts,
           if (!IsEligibleForPrefetch(offset, n)) {
             // Ignore status as Prefetch is not called.
             s.PermitUncheckedError();
+            if (usage_ == FilePrefetchBufferUsage::kTableOpenPrefetchTail) {
+              RecordTick(stats_, TABLE_OPEN_PREFETCH_TAIL_MISS);
+            }
```

Comment:
This is instrumenting in too many places. Can you do something like revert changes in this function, rename this function TryReadFromCacheUntracked, and use it as follows:  

```
... TryReadFromCache(...) {
  bool ret = TryReadFromCacheUntracked(...);
  if (usage_ == FilePrefetchBufferUsage::kTableOpenPrefetchTail) {
    if (ret) {
      RecordTick(stats_, TABLE_OPEN_PREFETCH_TAIL_HIT);
    } else {
      RecordTick(stats_, TABLE_OPEN_PREFETCH_TAIL_MISS);
    }
  }
  return ret;
}
```

Reply:
Good idea 💡

Reply:
Fixed.